### ISSUE TITLE
`transformation-runner-workspace/config.json` error when first running nori

### ISSUE THREAD

**Issue by taktran** [Financial-Times/nori#70]
When nori is first run from a `npm i -g nori` installation, we get the error:

```
{ Error: ENOENT: no such file or directory, open '/Users/jennifer.shepherd/.config/transformation-runner-workspace/config.json'
  errno: -2,
  code: 'ENOENT',
  syscall: 'open',
  path: '/Users/jennifer.shepherd/.config/transformation-runner-workspace/config.json' }
```

**Comment by jenniferemshepherd** [comment]
To solve the above I manually created the file config.json and the path for it.  This fixed that problem.

Now I'm getting:
```{ Error: ENOENT: no such file or directory, open '/Users/jennifer.shepherd/.config/transformation-runner-workspace/main-field-removal.nori.json'
  errno: -2,
  code: 'ENOENT',
  syscall: 'open',
  path: '/Users/jennifer.shepherd/.config/transformation-runner-workspace/main-field-removal.nori.json' }```

**Comment by quarterto** [comment]
hol up, `transformation-runner`? why's it trying to read that?

**Comment by quarterto** [comment]
okay i think this was fixed in `2.0.0-beta.4`
